backend/fms_core/viewsets/process_measurement.py

```diff
@@ -42,6 +42,12 @@ class ProcessMeasurementViewSet(viewsets.ModelViewSet, TemplateActionsMixin):
             "template": [{"description": "Template to transfer samples","file": SAMPLE_TRANSFER_TEMPLATE}],
             "importer": TransferImporter,
         },
+        {
```

Comment:
Move this template action to the sample ... will be more logical to have sample QC on sample and library QC on the library page.